src/activities/morse_code/FirstScreen.qml

```diff
@@ -0,0 +1,144 @@
+              qsTr('Each Morse code symbol represents either a text character (letter or numeral) or a prosign and is represented by a unique sequence of dots and dashes. ' +
+                   'The duration of a dash is three times the duration of a dot.' +
+                   ' To increase the speed of the communication, the code was designed so that the most common letters have the shorter sequences of dots and dashes.' + "\n" +
+                    'For example, the most common letter in English, the letter "E", has the shortest code, a single dot.')
+        fontSize: smallSize
+        fontSizeMode: Text.Fit
+        font.weight: Font.DemiBold
+        horizontalAlignment: Text.AlignJustify
+        verticalAlignment: Text.AlignJustify
+        wrapMode:  Text.WordWrap
+        anchors {
+            top: heading.bottom
+            topMargin: 5 * ApplicationInfo.ratio
+            right: parent.right
+            rightMargin: 5 * ApplicationInfo.ratio
+            left: introChar.right
+            leftMargin: 5 * ApplicationInfo.ratio
+            bottom: introChar.bottom
+
+        }
+        color: "#2a2a2a"
+    }
+    Item { // Just a margin
+        id:margin
+        width: 1
+        height: 15 * ApplicationInfo.ratio
+        anchors {
+            top: body_text1.bottom
+            left: introChar.right
+            right: parent.right
+            leftMargin: 5 * ApplicationInfo.ratio
+            rightMargin: 5 * ApplicationInfo.ratio
+        }
+    }
+
+    GCText {
+        id: bottom_text
+        text: qsTr("When you are ready, click on me and we will converse in Morse code.")
+        fontSize: regularSize
+        fontSizeMode: Text.Fit
+        font.weight: Font.Bold
+        width: parent.width * 0.75
+        color: "#2a2a2a"
+        horizontalAlignment: Text.AlignJustify
+        wrapMode:  Text.WordWrap
+        anchors {
+            top: margin.bottom
+            topMargin: 50 * ApplicationInfo.ratio
+            left: introChar.left
+            leftMargin: 10 * ApplicationInfo.ratio
+        }
+    }
+
+    Image {
+        id: introTux
+        source: "qrc:/gcompris/src/activities/morse_code/resource/tux.svg"
+        fillMode: Image.PreserveAspectFit
+        sourceSize.width: parent.width * 0.2
+        height: parent.height * 0.3
+        anchors {
+            top: bottom_text.top
+            left: bottom_text.right
+            right: parent.right
+            rightMargin: 5 * ApplicationInfo.ratio
+            leftMargin: 5 * ApplicationInfo.ratio
+        }
+        Behavior on scale { PropertyAnimation { duration: 100} }
+
+        MouseArea {
+            id: tux_click
+            anchors.fill: parent
+            hoverEnabled: true
+            onClicked: {
+                firstScreen.visible  = false
+                textInput.clear()
```

Comment:
qrc:/gcompris/src/activities/morse_code/FirstScreen.qml:138: TypeError: Property 'clear' of object QQuickTextInput_QML_150(0x2c18b50) is not a function
After clicking on Tux